specifyweb/frontend/js_src/lib/components/TreeView/index.tsx

```diff
@@ -104,10 +104,11 @@ function TreeView<SCHEMA extends AnyTree>({
   const baseUrl = `/api/specify_tree/${tableName.toLowerCase()}/${
     treeDefinition.id
   }`;
+  const doIncludeAuthorPref = getPref(`TaxonTreeEditor.DisplayAuthor`);
 
   const getRows = React.useCallback(
     async (parentId: number | 'null') =>
-      fetchRows(`${baseUrl}/${parentId}/${sortField}`),
+      fetchRows(`${baseUrl}/${parentId}/${sortField}/${doIncludeAuthorPref}`),
```

Comment:
This should be a query string parameter.
Consider refactoring to
```js
fetchRows(formatUrl(`${baseUrl}/${parentId}/${sortField}/`, {includeAuthor: doIncludeAuthorPref}))
```

Reply:
Fixed in https://github.com/specify/specify7/commit/2517904783158a6c2eea240956937f9f7f9368d8